packages/vote/fpc/parameters.go

```diff
@@ -0,0 +1,42 @@
+package fpc
+
+import "time"
+
+// Parameters define the parameters of an FPC instance.
+type Parameters struct {
+	// The lower bound liked percentage threshold at the first round. Also called 'a'.
+	FirstRoundLowerBoundThreshold float64
+	// The upper bound liked percentage threshold at the first round. Also called 'b'.
+	FirstRoundUpperBoundThreshold float64
+	// The liked percentage threshold used after the first round. Also called 'beta'.
+	SubsequentRoundsLowerBoundThreshold float64
+	// The amount of opinions to query on each round for a given vote context. Also called 'k'.
+	QuerySampleSize int
+	// The amount of rounds a vote context's opinion needs to stay the same to be considered final. Also called 'l'.
+	FinalizationThreshold int
+	// The amount of rounds for which to ignore any finalization checks for. Also called 'm'.
+	CoolingOffPeriod int
+	// The max amount of rounds to execute per vote context before aborting them.
+	MaxRoundsPerVoteContext int
+	// The max amount of time a query is allowed to take.
+	QueryTimeout time.Duration
+}
+
+// DefaultParameters returns the default parameters used in FPC.
+func DefaultParameters() *Parameters {
+	return &Parameters{
+		FirstRoundLowerBoundThreshold:       0.75,
+		FirstRoundUpperBoundThreshold:       0.85,
+		SubsequentRoundsLowerBoundThreshold: 0.33,
+		QuerySampleSize:                     10,
```

Comment:
Let's increase the `QuerySampleSize` to `21` for now.

Reply:
Changed